Library/Formula/keepassc.rb

```diff
@@ -31,13 +29,13 @@ def install
 
     man1.install Dir["*.1"]
 
-    bin.install Dir[libexec/'bin/*']
-    bin.env_script_all_files(libexec+'bin', :PYTHONPATH => ENV['PYTHONPATH'])
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec+"bin", :PYTHONPATH => ENV["PYTHONPATH"])
   end
 
   test do
-    # Fetching help is the only non-interactive action we can perform,
-    # and since interactive actions are un-scriptable, there nothing more we can do.
+    # Fetching help is the only non-interactive action we can perform, and since
+    # interactive actions are un-scriptable, there nothing more we can do.
     system "#{bin}/keepassc",  "--help"
```

Comment:
Test is failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/20479/version=yosemite/testReport/junit/brew-test-bot/yosemite/test_keepassc/